docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,533 @@
-  if you understand and accept the possible consequences and have exhausted all
-  other possibilities for recovery of your cluster.
+Using the ``unsafe-bootstrap`` command, you can unsafely override the voting
+configuration of a cluster as if performing another cluster-bootstrapping
+process. This operation is useful if there is at least one remaining
+master-eligible node, but it is not possible to restart a majority of the nodes.
 
-The sequence of operations for using this tool are as follows:
+If you have a choice of nodes on which to run the ``unsafe-bootstrap`` command,
+choose the one with the highest term. The node's terms increase with
+each election term. Whereas the node with the highest term is the
+most current node, consisting of the most current data.
 
-* Make sure you have really lost access to every one of the master-eligible nodes
-  in the cluster, and they cannot be repaired or recovered by moving their data
-  paths to healthy hardware.
+If there is more than one node with the same term, choose the one with the
+highest version.
 
-* Start a new cluster and verify that it is healthy. This cluster may comprise
-  one or more brand-new master-eligible nodes, or may be an unsafely-bootstrapped
-  cluster formed as described above.
+.. NOTE::
 
-* Stop all remaining data nodes.
+  Choosing the node with the highest term or version identifies the node with
+  the freshest cluster state what minimizes the quantity of data that might be
+  lost.
 
-* On each data node, run the ``crate-node detach-cluster`` tool as shown
-  below. Verify that the tool reported Node was successfully detached from
-  the cluster.
+Before running the ``unsafe-bootstrap`` command
+-----------------------------------------------
 
-* If necessary, configure each data node to discover the new cluster.
+Adhere to the following procedure before running the ``detach-cluster`` command:
 
-* Start each data node and verify that it has joined the new cluster.
+The ``unsafe-bootstrap`` command makes sure the node used to bootstrap the
+cluster is not running. All other master-eligible nodes must be stopped while
+this command is running.
 
-* Wait for all recoveries to have completed, and investigate the data in the
-  cluster to discover if any was lost during this process.
+  * Make sure you lost access to at least half of the cluster's master-eligible
+    nodes
 
-The message Node was successfully detached from the cluster does not mean that
-there has been no data loss, it just means that tool was able to complete its job.
+  * Make sure the lost nodes cannot be recovered by moving their data paths to
+    healthy hardware
 
-Parameters
-----------
+  * Choose a remaining master-eligible node to become the new elected master,
+    as described in :ref:`crate-node-repurpose-node-as-master`
 
-`repurpose`
-  Delete excess data when a node’s roles are changed.
+  * Run the ``unsafe-bootstrap`` command on the chosen node, as shown below.
```

Comment:
it seems unnecessarily complex to split this information over multiple sections

can we just combine it into one walkthrough?